webdriver-spec.html

```diff
@@ -3919,7 +3919,10 @@
  allow lookup of individual elements and collections of elements.
  Element retrieval searches are performed
  using pre-order traversal of the document’s nodes
- that match the provided selector’s expression.
+ that match the provided selector’s expression, and ignore the value
+ of the <a href="#dfn-page-load-strategy" class="internalDFN"
```

Comment:
I did see #438, which is about _implicit wait_ strategy not being mentioned in element retrieval. The element retrieval algorithm doesn’t explain what the driver should do when implicitly waiting for elements to appear in the DOM.